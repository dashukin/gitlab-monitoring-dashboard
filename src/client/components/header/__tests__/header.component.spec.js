import React from 'react';
import { shallow } from 'enzyme';
import Header from '../header.component';

describe('header', () => {
  describe('snapshots', () => {
    it('should match snapshot', () => {
      const wrapper = shallow(<Header />);

      expect(wrapper).toMatchSnapshot();
    });
  });
});
