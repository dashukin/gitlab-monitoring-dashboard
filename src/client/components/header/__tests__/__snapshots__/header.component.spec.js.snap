// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header snapshots should match snapshot 1`] = `
<WithStyles(ForwardRef(AppBar))>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(IconButton))
      aria-label="open drawer"
      color="inherit"
      edge="start"
    >
      <MenuIcon />
    </WithStyles(ForwardRef(IconButton))>
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Gitlab monitoring dashboard
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
